vhd/packer/install-dependencies.sh

```diff
@@ -501,6 +501,15 @@ for CSI_SECRETS_STORE_DRIVER_VERSION in ${CSI_SECRETS_STORE_DRIVER_VERSIONS}; do
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
+CLUSTER_PROPORTIONAL_AUTOSCALER_VERSIONS="
+1.7.1
+"
+for CLUSTER_PROPORTIONAL_AUTOSCALER_VERSION in ${CLUSTER_PROPORTIONAL_AUTOSCALER_VERSIONS}; do
+  CONTAINER_IMAGE="k8s.gcr.io/cluster-proportional-autoscaler-amd64:${CLUSTER_PROPORTIONAL_AUTOSCALER_VERSION}"
```

Comment:
@jchauncey @sozercan Are we interested in hosting MCR images for the cluster-proportional-autoscaler project?

Reply:
we have `mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler` in MCR.

Reply:
We have the following versions in mcr - 

```
"0.7.0",
"0.7.1",
"0.7.2",
"0.8.0",
"0.8.2",
"0.8.3",
"0.8.4",
"0.8.5",
"1.0.0",
"1.1.0",
"1.1.1",
"1.1.2",
"1.1.2-r2",
"1.1.2-r2_v0.0.5",
"1.2.0",
"1.2.0_v0.0.5",
"1.3.0",
"1.3.0_v0.0.5",
"1.4.0",
"1.5.0",
"1.6.0",
"1.7.0",
"1.7.1",
"1.7.1-hotfix.20200403",
```

Reply:
Terrific, thanks!

Anything to share on that April hotfix of 1.7.1? Should we use that instead of the official 1.7.1 release?

Reply:
I have no idea what was fixed in that version. You would have to contact Tony Xu to find out.

Reply:
10-4, thanks!